eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -1,13 +1,7 @@
 parameters:
-- name: PackageName
+- name: ArtifactName
```

Comment:
This will be a breaking change in all consumers of this template.

Reply:
I just want to make thinks clear because it seems to be causing issues that are easily missed. Where we pass artifact name thinking its package name then pass that down to another script that actually needs package name then it fails for JavaScript and Python. Am trying to use the specific names so that I can  identify where I need to make a conversion before passing it to another script.